Libraries/Components/Touchable/TouchableOpacity.js

```diff
@@ -164,16 +175,20 @@ class TouchableOpacity extends React.Component<Props, State> {
           this.props.onFocus(event);
         }
       },
-      onLongPress: event => {
-        if (this.props.onLongPress != null) {
-          this.props.onLongPress(event);
+      onKeyDown: event => {
+        if (this.props.onKeyDown != null) {
+          this.props.onKeyDown(event);
         }
       },
-      onPress: event => {
-        if (this.props.onPress != null) {
-          this.props.onPress(event);
+      onKeyUp: event => {
+        if (this.props.onKeyUp != null) {
+          this.props.onKeyUp(event);
         }
       },
+      validKeysDown: this.props.validKeysDown,
+      validKeysUp: this.props.validKeysUp,
+      onLongPress: this.props.onLongPress,
```

Comment:
can you keep the existing onlongpress and oppress and just bring the onkeyDown validkeydown stuff?

Reply:
Sure, not a bad idea to keep the null checks in place.